File: data/data.code.lua53.server/AckTestJsOk.lua
AckTestJsOk = {}
setmetatable(AckTestJsOk, {__index = _G})
local _ENV = AckTestJsOk


function new()
	local tab_obj = {}
	setmetatable(tab_obj, {__index = AckTestJsOk})
	tab_obj:init()
	return tab_obj
end

function init(self)
	self.u64 = 0
	self.i64 = 0
end


function encode(self)
	pack = packet.new()
	pack:writeu64(self.u64)
	pack:writei64(self.i64)
	return pack:encode(Msg.P_ACK_TEST_JS_OK)
end


function setU64(self, u64)
	self.u64 = u64
end

function setI64(self, i64)
	self.i64 = i64
end

File: data/data.code.php.server/AckTestJsOk.php
<?php
namespace protocol;

use network\Packet;


class AckTestJsOk
{
	private $u64 = 0;
	private $i64 = 0;


	public function encode()
	{
		$packet = new Packet();
		$packet->writeU64($this->u64);
		$packet->writeI64($this->i64);
		return $packet->encode(Msg::$P_ACK_TEST_JS_OK);
	}


	public function setU64($u64)
	{
		$this->u64 = $u64;
	}

	public function setI64($i64)
	{
		$this->i64 = $i64;
	}

}
